waiter/src/waiter/utils.clj

```diff
@@ -544,3 +544,11 @@
   (if (instance? ExceptionInfo e)
     (ex-info (.getMessage e) (merge (ex-data e) m) (.getCause e))
     (ex-info (.getMessage e) m (.getCause e))))
+
+(defmacro tryv
+  "Like try, but [result exception]"
```

Comment:
Why not return a map with `:result` and `:exception`? We don't seem to be using both slots of the vector at the same time.